v1.1/create-security-certificates-openssl.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Create Security Certificates
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
+toc: false
+---
+
+A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication, which requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, or use [`openssl` commands](https://wiki.openssl.org/index.php/).
+
+<div class="filters filters-big clearfix">
+  <a href="create-security-certificates.html"><button style="width:28%" class="filter-button">Use Cockroach Cert Commands</button>
+  <button style="width:28%" class="filter-button current"><strong>Use OpenSSL Commands</strong></button></a>
```

Comment:
Can you start the `<strong>` emphasis at `OpenSSL`? Same on non-openssl page.

The button text now feels too verbose. Could we try:
- Use **cockroach cert**
- Use **openssl**